config/main.py

```diff
@@ -3916,6 +3916,172 @@ def del_user(db, user):
             click.echo("Restart service snmp failed with error {}".format(e))
             raise click.Abort()
 
+#
+# 'bmp' group ('config bmp ...')
+#
+
+@config.group()
+@click.pass_context
+def bmp(ctx):
+    """BMP-related configuration"""
+
+#
+# 'enable' subgroup ('config bmp enable ...')
+#
+@bmp.group()
+@click.pass_context
+def enable(ctx):
+    """Enable BMP table dump """
+
+#
+# 'bgp-neighbor-table' command ('config bmp enable bgp-neighbor-table')
+#
+@enable.command('bgp-neighbor-table')
+@click.pass_context
+def enable_bgp_neighbor_table(ctx):
+    """enable bgp-neighbor-table sessions
+       In the case of Multi-Asic platform, corresponding database instance of neighbor will be operated.
+    """
+    log.log_info("'bmp enable bgp-neighbor-table' executing...")
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    try:
+        bmp_table = config_db.get_table('BMP')
+        if not bmp_table:
+            bmp_table =  {'table': {'bgp_neighbor_table': 'true'}}
+        else:
+            bmp_table['table']['bgp_neighbor_table'] = 'true'
```

Comment:
>bmp_table['table']['bgp_neighbor_table'] = 'true' [](http://example.com/codeflow?start=12&length=49)

Make sure this line is covered in unit test and coverage result.

Reply:
fixed.